demo/README.md

```diff
@@ -36,19 +36,19 @@ or use checkpoint url from `configs/` to directly load corresponding checkpoint,
 1. Recognize a video file as input by using a TSN model on cuda by default.
 
     ```shell
-    # The demo.mp4 and label_map.txt are both from Kinetics-400
+    # The demo.mp4 and k400_label_map.txt are both from Kinetics-400
```

Comment:
`k400_label_map` -> `label_map_k400`, so that label map files of different datasets are grouped together naturally